addons/project/models/project.py

```diff
@@ -2003,8 +2004,8 @@ def create(self, vals_list):
                 vals['date_assign'] = fields.Datetime.now()
                 if not project_id:
                     user_ids = self._fields['user_ids'].convert_to_cache(vals.get('user_ids', []), self)
-                    if self.env.user.id not in user_ids:
-                        vals['user_ids'] = [Command.set(list(user_ids) + [self.env.user.id])]
+                    if not is_superuser:
```

Comment:
Why not keeping the previous condition as well?
```suggestion
                    if not is_superuser and self.env.user.id not in user_ids:
```

Reply:
Hello @xavierbol my bad :smiling_face_with_tear: 

changes are applied.

Thank you :slightly_smiling_face: